spec/features/tasks_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'feature_helper'
+
+describe "Tasks" do
+
+  before(:each) do
+    # FIXME - Having to set this really high for the 'adds a task to a story
+    # spec'.  Need to work on making it more responsive.
+    Capybara.default_max_wait_time = 10
+    sign_in user
+  end
+
+  let(:user)        { create :user, :with_team, email: 'user@example.com', password: 'password' }
+  let(:project)     { create(:project, name: 'Test Project', users: [user], teams: [user.teams.first] ) }
+
+  let!(:story) do
+    create :story,  title: 'Test Story',
+                    state: 'started',
+                    project: project,
+                    requested_by: user
+  end
+
+  describe "full story life cycle" do
+
+    it "adds a task to a story", js: true do
+      visit project_path(project)
+
+      within('#in_progress .story') do
+        find('.story-title').trigger('click')
+        fill_in 'task', with: 'Adding a new task'
+        click_on 'Add task'
+      end
+
+      task_element = find('#in_progress .story .tasklist .task')
+      expect(task_element).to have_content('Adding a new task')
+
+    end
+
+    it "deletes a task from a story", js: true do
+      create :task, story: story,
+                    name: 'Delete me please'
```

Comment:
This could be just one line.
And the same comment, if you decide to add `(`, don't forget here.